index.js

```diff
@@ -19,12 +19,13 @@ const linkify = (href, options) => createHtmlElement({
 // Get DOM node from HTML
 const domify = html => document.createRange().createContextualFragment(html);
 
-const getAsString = (string, options) => string.replace(urlRegex(), match => linkify(match, options));
+const getAsString = (string, options) => string.replace(urlRegex(), (match, _, offset) => offset + match.length + 1 === string.length && string.charAt(string.length - 1) === '…' ? match : linkify(match, options));
```

Comment:
Use `String#endsWith('…')`

Reply:
I don't think this is the right logic. It should be:

- if next item starts with …, then return match as is

I think the currently logic skips linkifying: `a https://example.com like this…` simply because the whole string ends with …